people.md

```diff
@@ -216,4 +216,14 @@ Undergraduate Researcher, Brown UTRA Award Recipient
 +++
 
 
+---
+:column: col-4
+
+Yemi Fasina
+^^^
+Undergraduate Researcher
+
++++
+
+
```

Comment:
this is in the alumni section, can you move it up?

Reply:
ahh I did not notice that. I just moved it up.